python/test/t_SubsetSampling_R-S.py

```diff
@@ -105,8 +105,9 @@
 
 
 # check that the event sample is right
-inputEventSample = mySS.getEventInputSample()
-outputEventSample = mySS.getEventOutputSample()
+Ns = mySS.getStepsNumber()
+inputEventSample = mySS.getInputSample(Ns - 1, 1)
+outputEventSample = mySS.getOutputSample(Ns - 1, 1)
```

Comment:
If a variable name has an upcase first letter, it is assumed that this is a class name: see [PEP8](https://peps.python.org/pep-0008/#function-and-variable-names)
```suggestion
ns = mySS.getStepsNumber()
inputEventSample = mySS.getInputSample(ns - 1, 1)
outputEventSample = mySS.getOutputSample(ns - 1, 1)
```